buildroot-external/patches/occu/0093-WebUI-AllowAdminUserChanges.patch

```diff
@@ -0,0 +1,41 @@
+--- occu/WebUI/www/rega/esp/system.fn.orig
++++ occu/WebUI/www/rega/esp/system.fn
+@@ -1164,7 +1164,7 @@
+     oUser = dom.GetObject( system.GetVar("userid") );
+     if( oUser )
+     {
+-      if( !oUser.Unerasable() )
++      if( !oUser.Unerasable() || oUser.ID() == 1004 )
+       {
+         object oFavorites = dom.GetObject( ID_FAVORITES );
+         if( oFavorites )
+--- occu/WebUI/www/rega/pages/msg/userAccountConfigAdmin.htm.orig
++++ occu/WebUI/www/rega/pages/msg/userAccountConfigAdmin.htm
+@@ -25,7 +25,7 @@
+       if(userToTest.Name() == "Admin") 
+       {
+         sDisabled = "disabled";
+-        sSuperAdminDisabled = "disabled";
++        !sSuperAdminDisabled = "disabled";

```

Comment:
This allows the user `Admin` to rename its username.

To keep the difference to the original file small, I only commented this out.
_The whole function could be reworked, but this will lead to large differences to the original OCCU file_